### PR TITLE
Updated Doc: Get Started - React.js

### PR DESCRIPTION
**Pull request by Akash-Patil**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/docs/blob/master/CONTRIBUTING.md) before creating this PR.

##### Description of change

- Updated the `Get Started - React.js` Document. Made changes to the code snippets and added extra features pertaining to the newly released `LoginRadius React SDK`.

### CODE REVIEW

File: content/tutorial/react/index.md [hunk 1]
```diff
@@ -90,11 +130,33 @@ Navigate your Register or Login links or buttons to the following URLs:
...
+    );
+  } else {
+    return <button onClick={loginWithRedirect}>Log in</button>;
+  }
+};       
+``` 
+
 > return_url can be your website, frontend app, or backend server url where you are handling the access token.
 >
 > For this tutorial, set the `return_url` to point to the `/login` subdomain of your application. For example, in the local React instance, it can point to `http://localhost:3000/login`. This way, after logging in through the Auth Page (IDX) Login page, it will redirect the user to the Login component implemented in below [step](#retrieve-user-data-using-access-token).
```
Comment (mohammed786) [inline content/tutorial/react/index.md:157]: Do we need this paragraph?

File: content/tutorial/react/index.md [hunk 2]
```diff
@@ -90,11 +130,33 @@ Navigate your Register or Login links or buttons to the following URLs:
...
 
-## Retrieve User Data using Access Token
+## Retrieve User Data 
 
 > Once the authentication is done using Auth Page (IDX), the default script of LoginRadius sends an access token in the query string as a token parameter with the return_url. The return_url should be your application's web page where you would like to receive the access token.
 > The following is an example of the access token in the query string with the Return URL:
```
Comment (mohammed786) [inline content/tutorial/react/index.md:162]: We don't need to use this method right, we can get the accessToken via the `getAccessTokenSilently` method of `useLRAuth`

File: content/tutorial/react/index.md [hunk 3]
```diff
@@ -142,53 +204,23 @@ Navigate your Register or Login links or buttons to the following URLs:
...
+    <div>
+      <span style={{ whiteSpace: "pre-wrap", textAlign: "left" }}>
+        {JSON.stringify(user, null, 4)}
+      </span>
+    </div>
+     );
+  } 
+  export default withAuthenticationRequired(Login, {
+  onRedirecting: () => <Loading />, returnTo: '/login',
+  });
```
Comment (mohammed786) [inline content/tutorial/react/index.md:221]: Here the component name should be **Profile** i think?

File: content/tutorial/react/index.md [hunk 4]
```diff
@@ -33,16 +33,56 @@ In your LoginRadius Dashboard, navigate to **<a href="https://dashboard.loginrad
...
-This example uses a sample app based on the Create React App (CRA) boilerplate. For directions on how to Create React App, you can reference <a href="https://reactjs.org/docs/create-a-new-react-app.html" target="_blank">here.</a>
+This example uses a sample app based on the Create React App (CRA) boilerplate. For directions on how to Create React App, you can reference <a href="https://reactjs.org/docs/create-a-new-react-app.html" target="_blank">here.</a> 
 
 Once the CRA boilerplate is set up, follow these steps:
 
 - Navigate to the project root and install `react-router-dom`:
 
   `npm install react-router-dom`
 
+- We will also be using the [LoginRadius React SDK](https://github.com/LoginRadius/loginradius-react) to implement different functionalities.
```
Comment (mohammed786) [inline content/tutorial/react/index.md:54]: This line should come under the header **Install LoginRadius React SDK**. And explain in short what does login radius react SDK does. Something like this 
```
The LoginRadius React SDK uses [React Context](https://reactjs.org/docs/context.html) to manage the authentication state of your users. One way to integrate LoginRadius with your React app is to wrap your root component with an `LRAuthProvider` that you can import from the SDK.
```

**Review by mohammed786 — CHANGES_REQUESTED** [review]
Kindly check the comments,

File: content/tutorial/react/index.md (hunk 3 above)
Comment (Akash-Patil) [reply content/tutorial/react/index.md:215]: Component is named Login for consistency.

**Review by mohammed786 — APPROVED** [review]

File: content/tutorial/react/index.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ path: "/tutorial/react"
 
 # Get Started - React.js
 
-The tutorial lets you implement LoginRadius user registration, login, and view profile on your React.js based application.
+The tutorial lets you implement LoginRadius user registration, login, and view profile on your React.js based application. It amkes use of the LoginRadius React SDK.
```
Comment (abhir9) [inline content/tutorial/react/index.md:10]: amkes -> makes

**Review by pareek-narendra — APPROVED** [review]

**Pull request merged**